components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -282,7 +282,7 @@ function WorkspaceImageButton(props: {
         };
     }
 
-    const image = props.settings?.defaultWorkspaceImage ?? props.defaultWorkspaceImage ?? "";
+    const image = props.settings?.defaultWorkspaceImage || props.defaultWorkspaceImage || "";
```

Comment:
This one is potentially breaking for old dashboard against new server, but affect is not significant and resolved by the page reload.

Ideally we need to deliver Huiwen's PR to stop worrying about it.

Reply:
We discussed that a risk is pretty low, given that update is very rare operation and we deploy to cells in non working hours. Anyway we wait today for https://github.com/gitpod-io/gitpod/pull/19042 and if it takes longer then we land this one. cc @mustard-mh

Reply:
@mustard-mh I thinkg there is not problem actually, JSON-RPC is backward compatible, i.e. returns undefined value and old dashboard is going to use old gRPC types with optional, so it should not break, does it make sense?